news/2022-11-30-new-featured-artist-noisestorm.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: "New Featured Artist: Noisestorm"
+date: 2022-11-30 11:00:00 +0000
+---
+
+🦀🦀🦀
+
+![](https://assets.ppy.sh/artists/324/header.jpg)
+
+Legendary producer **Noisestorm** has been dominating the electronic scene for more than a decade, and starting today, they're going to dominate osu!'s Featured Artist listing too.
+
+While his discography is full of classics, **Noisestorm**'s most recent claim to fame is the iconic tune *Crab Rave*, which has spread to practically every online community since its release in 2018. osu! is no exception of course, with a [beatmap of Crab Rave](https://osu.ppy.sh/beatmapsets/891596) amassing millions of plays since its release.
```

Comment:
```suggestion
While his discography is full of classics, **Noisestorm**'s most recent claim to fame is the iconic tune *Crab Rave*, which has spread to practically every online community since its release in 2018. osu! is no exception of course, with a [beatmap of *Crab Rave*](https://osu.ppy.sh/beatmapsets/891596) amassing millions of plays since its release.
```